server/models.py

```diff
@@ -42,32 +50,40 @@ class Player(db.Model):
     execution_id: Mapped[int] = mapped_column(
         ForeignKey("execution.id"), nullable=False
     )
-    location_id: Mapped[int] = mapped_column(ForeignKey("location.id"), nullable=False)
+    location_id: Mapped[int] = mapped_column(
+        ForeignKey("location.id"), nullable=False)
     role_id: Mapped[int] = mapped_column(ForeignKey("role.id"), nullable=False)
     alerted: Mapped[bool] = mapped_column(nullable=False)
     activation_delay_sec: Mapped[int] = mapped_column(nullable=False)
 
+    execution: Mapped["Execution"] = relationship(back_populates="players")
+
```

Comment:
Why is this needed? Is this also a column in the DB or just a reference in the ORM-object?